blog/_posts/2018-06-13-accessible-scala.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog-detail
+post-type: blog
+by: Guillaume Massé
+title: Introducing Accessible Scala
+---
+
+  Have you found yourself where you had to verbally describe code? For example, while pair-programming
+or talking about work during lunch. Verbally describing code is not an easy task, fortunately for you the
+Scala Center is working on this. [SCP-016]: Accessible Scala, was submitted to provide industry-leading support
+for blind and partially-sighted developers. It's also an opportunity to formalize a spoken Scala. For example,
```

Comment:
Again, we did not intend to "formalize spoken Scala." Can you please remove this?